packages/traceability-schemas/src/generators/Event.js

```diff
@@ -0,0 +1,27 @@
+const faker = require('faker');
+const { getOrganization } = require('./Organization');
+const { getPlace } = require('./Place');
+const { getProduct } = require('./Product');
+
+const getEvent = () => {
+    const eventType = 'commission';
+    const eventId = '12345';
+    const actor = [getOrganization(), getOrganization()];
```

Comment:
likely these helpers are injecting `@context`... please make sure we are not testing them in example data.

Reply:
theres two ways to fix this in my mind either:
 * we need a parameter to these helpers that indicates whether to inject context or not (my preference)
 * we need to strip context after its generated in each of these helpers